dvc/system.py

```diff
@@ -146,10 +146,9 @@ def reflink(source, link_name):
 
     @staticmethod
     def _getdirinfo(path):
-        import ctypes
-        from ctypes import c_void_p, c_wchar_p, Structure, WinError, POINTER
-        from ctypes.wintypes import DWORD, HANDLE, BOOL
         from win32file import (
```

Comment:
do we need to insstall it - setup.py? wheels, whatnot? what is the license?

Reply:
@shcheklein win32file was here for a long time, no additional actions required.

Reply:
👍